From 4503c2cbd3981ba1b50ec7ce287ab48c772b3a7f Mon Sep 17 00:00:00 2001
From: Dave Mielke <Dave@Mielke.cc>
Date: Sat, 15 Oct 2022 15:20:38 -0400
Subject: [PATCH] shell prologue: runProgramTerminationCommands used a bash
 command. (dm)

Signed-off-by: James Hilliard <james.hilliard1@gmail.com>
[james.hilliard1@gmail.com: backport from upstream commit
4503c2cbd3981ba1b50ec7ce287ab48c772b3a7f]
---
 brltty-prologue.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/brltty-prologue.sh b/brltty-prologue.sh
index 05b817742..6d1faf983 100644
--- a/brltty-prologue.sh
+++ b/brltty-prologue.sh
@@ -102,7 +102,7 @@ runProgramTerminationCommands() {
    while [ "${programTerminationCommandCount}" -gt 0 ]
    do
       set -- $(getVariable "programTerminationCommand${programTerminationCommandCount}")
-      let "programTerminationCommandCount -= 1"
+      programTerminationCommandCount=$((programTerminationCommandCount - 1))
 
       local process="${1}"
       local directory="${2}"
-- 
2.34.1

